README.md

```diff
@@ -10,6 +10,10 @@ This solves the problem when only **mono** output is recognized for Nari.
 
 ### PipeWire
 
+If you use PipeWire, I've created an install script. Use the install-pipewire.sh script by ``cd`` into the ``razer-nari-pulseaudio-profile`` folder and run ``sudo chmod +x install-pipewire.sh && ./install-pipewire.sh``
+
+Or, you can make the changes yourself manuallY:
```

Comment:
Typo here
```suggestion
Or, you can make the changes yourself manually:
```

However, I think this whole line can be removed

Reply:
Agreed it can be removed. I'm fairly new at this and in fact this is my first pull request lol. Do I make changes or you?